### ISSUE TITLE
Some new Check helpers create obtuse error messages.

### ISSUE THREAD

**Issue by mramato** [AnalyticalGraphicsInc/cesium#4882]
All error messages generated by `Check` should contain the name of the parameter they apply to in a clear maner.  Some of the helper functions, like `Check.numeric.maximum`, generate messages similar to:

`Expected 5 to be at most 4` 

This doesn't help the developer much at all.  It would be better to generate something like:

`Expected options.north to be less than 4 but its values was 5`

We should do this ASAP since we are in the process of switching over the codebase to use `Check`.

CC @lilleyse

**Comment by lilleyse** [comment]
@erikmaarten would you like to make these changes to `Check`.

@rahwang once a fix is in would you mind combing through the `Check` related PRs opened recently and fix them up?

More discussion here: https://github.com/AnalyticalGraphicsInc/cesium/pull/4881

**Comment by rahwang** [comment]
@lilleyse Sure thing! I'll clean up once the fix is done.

**Comment by erikmaarten** [comment]
That was an oversight on my part.

I've opened #4885 to change these methods in `Check`. I'm not sure what's the easiest way to do the remaining fixes here, but @rahwang if you fork my Cesium fork, branch off from `improve-check-errors` and make a PR with the fixes back into my `erikmaarten:improve-check-errors` that should work. Branching off from my branch and making a new PR into Cesium master, replacing my PR, might be easier.

Thanks in advance!
